packages/ui/src/message-pane-input/MessagePaneInput.jsx

```diff
@@ -86,7 +91,7 @@ const MessagePaneInput = ({ onSendMessage, users, onAttachFile }) => {
   const [showEmoji, setShowEmoji] = useState(false);
   const [suggestions, setSuggestions] = useState(users || mentions);
   const [suggestionsOpen, setSuggestionsOpen] = useState(false);
-  // console.log("field", editorState)
+  // console.log("field", editorState);
```

Comment:
If the code isn't being used, instead of commenting it out you could do away with it entirely